docs/index.rst

```diff
@@ -92,12 +123,13 @@ Technical Documentation
 
 * `LSST Camera verification testing and characterization <https://ui.adsabs.harvard.edu/abs/2024SPIE13096E..1SR/abstract>`_, Roodman et al. (2024; SPIE, `doi:10.1117/12.3019698 <https://doi.org/10.1117/12.3019698>`_)
 * `LSST Camera focal plane optimization <https://ui.adsabs.harvard.edu/abs/2024SPIE13103E..0WU/abstract>`_, Utsumi et al. (2024; SPIE, `doi:10.1117/12.3019117 <https://doi.org/10.1117/12.3019117>`_)
-* `Integrating the LSST camera <https://ui.adsabs.harvard.edu/abs/2024SPIE13096E..1OL/abstract>`_ Lange et al. (2024; SPIE `doi:10.1117/12.3019302 <https://doi.org/10.1117/12.3019302>`_)
-* `LSST Camera Electro-Optical Test (Run 7) Results <https://sitcomtn-148.lsst.io>`_ Antilogus et al. (2025, NSF-DOE Vera C. Rubin Observatory Commissioning Technical Note 148)
-* `Design and development of the 3.2 gigapixel camera for the Large Synoptic Survey Telescope <https://ui.adsabs.harvard.edu/abs/2010SPIE.7735E..0JK/abstract>`_ Kahn et al. (2010; SPIE, `doi:10.1117/12.857920 <https://doi.org/10.1117/12.857920>`_)
-* `The LSST camera overview: design and performance <https://ui.adsabs.harvard.edu/abs/2008SPIE.7014E..0CG/abstract>`_ Gilmore et al. (2008; SPIE, `doi:10.1117/12.789947 <https://doi.org/10.1117/12.789947>`_)
-* `The LSST Camera system overview <https://ui.adsabs.harvard.edu/abs/2006SPIE.6269E..0CG/abstract>`_ Gilmore et al. (2006; SPIE, `doi:10.1117/12.673236 <https://doi.org/10.1117/12.673236>`_)
-
+* `Integrating the LSST camera <https://ui.adsabs.harvard.edu/abs/2024SPIE13096E..1OL/abstract>`_, Lange et al. (2024; SPIE `doi:10.1117/12.3019302 <https://doi.org/10.1117/12.3019302>`_)
+* `LSST Camera Electro-Optical Test (Run 7) Results <https://sitcomtn-148.lsst.io>`_, Antilogus et al. (2025, NSF-DOE Vera C. Rubin Observatory Commissioning Technical Note 148)
+* `Design and development of the 3.2 gigapixel camera for the Large Synoptic Survey Telescope <https://ui.adsabs.harvard.edu/abs/2010SPIE.7735E..0JK/abstract>`_, Kahn et al. (2010; SPIE, `doi:10.1117/12.857920 <https://doi.org/10.1117/12.857920>`_)
+* `The LSST camera overview: design and performance <https://ui.adsabs.harvard.edu/abs/2008SPIE.7014E..0CG/abstract>`_, Gilmore et al. (2008; SPIE, `doi:10.1117/12.789947 <https://doi.org/10.1117/12.789947>`_)
+* `The LSST Camera system overview <https://ui.adsabs.harvard.edu/abs/2006SPIE.6269E..0CG/abstract>`_, Gilmore et al. (2006; SPIE, `doi:10.1117/12.673236 <https://doi.org/10.1117/12.673236>`_)
+* `LCA-13381: Detector Plane Layout <https://zenodo.org/records/15133467>`_ , M. Nordby, 2020
```

Comment:
Please remove this line. CTN-001 is good enough.

Reply:
removed